modules/ROOT/nav.adoc

```diff
@@ -155,7 +163,7 @@
  ** xref:manage:troubleshoot/core-files.adoc[Core Files]
 
 .Develop
-* xref:sdk:development-intro.adoc[Developer's Intro]
+//* xref:sdk:development-intro.adoc[Developer's Intro]
```

Comment:
Let's put the tutorial here for now, at least until we've had a chance to merge the developer how-to guides. I've also given the tutorial a short name for the purposes of the Navigation file. Finally, I've reinstated `development-intro.adoc` for now, although I have renamed it for the purposes of the navigation. We can remove this file for good when we've had a chance to merge the developer how-to guides.

```suggestion
* xref:tutorials:couchbase-tutorial-student-records.adoc[Couchbase Tutorial]
  ** xref:tutorials:install-couchbase-server.adoc[]
  ** xref:tutorials:buckets-scopes-and-collections.adoc[]
  ** xref:tutorials:java-tutorial/install-couchbase-java-sdk.adoc[]
  ** xref:tutorials:java-tutorial/creating-the-students-collection.adoc[]
  ** xref:tutorials:java-tutorial/creating-the-courses-collection.adoc[]
  ** xref:tutorials:java-tutorial/retrieving-documents.adoc[]
  ** xref:tutorials:java-tutorial/adding-course-enrollments.adoc[]
* xref:sdk:development-intro.adoc[Key-Value Operations]
```